src/releasers/node.ts

```diff
@@ -119,9 +117,28 @@ export class Node extends ReleasePR {
     };
   }
 
+  // Always prefer the package.json name
+  async getPackageName(): Promise<PackageName> {
+    if (!this.pkgJsonContents) {
+      this.pkgJsonContents = await this.getPkgJsonContent();
+    }
+    if (!this._packageName) {
+      const pkg = JSON.parse(this.pkgJsonContents.parsedContent);
+      this.packageName = this._packageName = pkg.name ?? this.packageName;
+    }
+    return {
+      name: this.packageName,
+      refSafeName: () =>
```

Comment:
If I'm understanding `refSafeName` properly, it codifies dropping the `@pkg/` prefix from a branch name?

Reply:
yeah, but I think I'll change it to `getComponent()` per a discussion with @chingor13

> In this refactor I tried to steer away as much as possible from functionality refactoring or re-naming. The one exception is that darned quagmire of (packagePrefix vs packageName) x (set via configuration vs releaser knows how to get it): there was a nasty tangle of responsibility leaking that I needed to cleanup to normalize the configuration. Maybe `PackageName.refSafeName()` is better named `PackageName.getComponent()` - that's more inline with the domain vocabulary we've chosen and being safe to use as/in a git ref is just a requirement of a component rather than the name